### PR TITLE
add code description

### PR DESCRIPTION
**Pull request by zjj2wry**
add code description

### CODE REVIEW

File: tensorflow/examples/learn/iris_custom_decay_dnn.py [hunk 1]
```diff
@@ -11,6 +11,8 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+"""Example of DNNClassifier for Iris plant dataset,with exponential decay"""
```
Comment (caisq) [inline tensorflow/examples/learn/iris_custom_decay_dnn.py]: nit: Per Google Python Style (https://google.github.io/styleguide/pyguide.html), there needs to be
1) A space after the comma,https://google.github.io/styleguide/pyguide.html
2) A period after the last word.

Also make sure that the line is <=80 chars wide as you make further changes.

**Review by caisq — DISMISSED** [review]

File: tensorflow/examples/learn/iris_custom_decay_dnn.py (hunk 1 above)
Comment (zjj2wry) [inline tensorflow/examples/learn/iris_custom_decay_dnn.py]: @caisq think you very much, it is helpful. 💯

**Comment by drpngx** [comment]
Jenkins, test this please.

**Review by drpngx — APPROVED** [review]

**Comment by drpngx** [comment]
Jenkins, test this please.

**Pull request merged**